content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -262,6 +262,29 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+### config.kubernetes.io/depends-on
+
+Type: Annotation
+
+Example: `config.kubernetes.io/depends-on:`
+
+```yaml
+  annotations:
+  config.kubernetes.io/depends-on: |
+    /namespaces/default/Pod/pod-a,
+    /namespaces/default/Pod/pod-c
+
+```
+
+Used on: All objects
+
+This annotation is used on the target object to be modified to configure explicit dependency ordering. 
+The annotation helps to determine the apply order, and dependency reconciliation while applying and deletion and prevents accidental premature deletion.
+
+For example If `config.kubernetes.io/depends-on: /namespaces/default/Pod/pod-a` annotation is added to `pod-b`,  it identifies that pod-a must be successfully applied prior to pod-a actuation.
+
+For more details, see [depends on](https://kpt.dev/reference/annotations/depends-on/) in the KPT documentation. 
```

Comment:
I know I also first found this from KPT, but I wonder if a better permalink is https://github.com/kubernetes-sigs/cli-utils/blob/0b156cb0425fdb29a436d13f840a39039558c10e/README.md#explicit-dependency-ordering